src/_includes/core-nodes/complete-use-case.md

```diff
@@ -0,0 +1,268 @@
+
+# Example
+
+1. In the example flow below, we have a WebSocket server, the Inject node sends data to the WebSocket server, and upon successful transmission to the WebSocket server, a Complete node handles the event.
+
+{% renderFlow %}
+
+[
+    {
+        "id": "75e4c8a97e30b108",
+        "type": "inject",
+        "z": "1027b41ed961cbc5",
+        "name": "send data",
+        "props": [
+            {
+                "p": "payload"
+            }
+        ],
+        "repeat": "",
+        "crontab": "",
+        "once": false,
+        "onceDelay": 0.1,
+        "topic": "",
+        "payload": "",
+        "payloadType": "date",
+        "x": 400,
+        "y": 240,
+        "wires": [
+            [
+                "85383be37566ea36"
+            ]
+        ]
+    },
+    {
+        "id": "85383be37566ea36",
+        "type": "websocket out",
+        "z": "1027b41ed961cbc5",
+        "name": "websocket server",
+        "server": "65bb0cfe75e94539",
+        "client": "",
+        "x": 690,
+        "y": 240,
+        "wires": []
+    },
+    {
+        "id": "c51c076d5d15b727",
+        "type": "complete",
+        "z": "1027b41ed961cbc5",
+        "name": "complete",
+        "scope": [
+            "85383be37566ea36"
+        ],
+        "uncaught": false,
+        "x": 380,
+        "y": 360,
+        "wires": [
+            [
+                "206f2f78564b40d8"
+            ]
+        ]
+    },
+    {
+        "id": "206f2f78564b40d8",
+        "type": "debug",
+        "z": "1027b41ed961cbc5",
+        "name": "debug 1",
+        "active": true,
+        "tosidebar": true,
+        "console": false,
+        "tostatus": false,
+        "complete": "false",
+        "statusVal": "",
+        "statusType": "auto",
+        "x": 740,
+        "y": 360,
+        "wires": []
+    },
+    {
+        "id": "3a3c5d99ecf56c4a",
+        "type": "comment",
+        "z": "1027b41ed961cbc5",
+        "name": "Sending data to websocket server",
+        "info": "",
+        "x": 520,
+        "y": 180,
+        "wires": []
+    },
+    {
+        "id": "126eac07e1f41280",
+        "type": "comment",
+        "z": "1027b41ed961cbc5",
+        "name": "Upon successful data transmission to the WebSocket server, the Complete node handles the event.",
+        "info": "",
+        "x": 560,
+        "y": 300,
+        "wires": []
+    },
+    {
+        "id": "65bb0cfe75e94539",
+        "type": "websocket-listener",
+        "path": "/ws/response",
+        "wholemsg": "false"
+    }
+]
+
+{% endrenderFlow %}
+
+2. In this example, we simulate a time-consuming task using setTimeout in a function named "Long running process". After completion, we mark success with `node.done()` using a Complete node. Then, perform further action we process the result and assign it to `msg.payload` for further use.
+
+{% renderFlow %}
+
+[
+    {
+        "id": "f3567f52f2c9f98b",
+        "type": "function",
```

Comment:
Why not make a get request? Network is a much better example than using a function node

Reply:
Okay, Zj, I will update it with a GET request.

Reply:
done !